server/node.go

```diff
@@ -170,6 +170,12 @@ func NewNode(ctx storage.StoreContext) *Node {
 	}
 }
 
+// context returns a base context for Node-level use. It is shared with
+// the local sender and contains NodeID and ClusterID (once known).
+func (n *Node) context() context.Context {
```

Comment:
I think we should probably add a context.Context parameter to KVSender.Send and pass it through.